### PR TITLE
docs: fix broken link

### PR DESCRIPTION
**Pull request by dito**

### CODE REVIEW

File: docs/reference/builder.md [hunk 1]
```diff
@@ -608,7 +608,7 @@ FROM [--platform=<platform>] <image>[@<digest>] [AS <name>]
 ```
 
 The `FROM` instruction initializes a new build stage and sets the
-[*Base Image*](../../glossary/#base-image) for subsequent instructions. As such, a
+[*Base Image*](https://docs.docker.com/glossary/#base_image) for subsequent instructions. As such, a
```
Comment (thaJeztah) [inline docs/reference/builder.md:611]: I guess this was originally linking to https://github.com/docker/cli/blob/master/docs/reference/glossary.md#base-image, but that page is not published on docs.docker.com. Perhaps we should remove it here

**Comment by thaJeztah** [comment]
Actually, if you have a chance, could you update your DCO sign-off to use your real name, not your GitHub handle (https://github.com/docker/cli/blob/master/CONTRIBUTING.md#sign-your-work)?

```
Signed-off-by: dito <itodaisuke00@gmail.com>
```

**Comment by thaJeztah** [comment]
ping @dito ^^

**Comment by dito** [comment]
thanks! I updated my DCO.

**Pull request merged**